api/GovStack_Registration_BB_API_workflow-0.0.1-resolved.json

```diff
@@ -0,0 +1,525 @@
+					},
+					"code": {
+						"type": "string",
+						"example": "4006"
+					},
+					"message": {
+						"type": "string",
+						"example": "Something is wrong with the data, please check one more time."
+					},
+					"localizedMessage": {
+						"type": "string",
+						"example": "error"
+					}
+				}
+			},
+			"ErrorDetailDto": {
+				"type": "object",
+				"properties": {
+					"message": {
+						"type": "string",
+						"example": "Something is wrong with the data, please check one more time."
+					},
+					"field": {
+						"type": "string",
+						"example": "42962de0-bdb2-11ed-9397-0242ac120004"
+					},
+					"value": {
+						"type": "string",
+						"example": "test ABC"
+					}
+				}
+			},
+			"ErrorsListDto": {
+				"type": "object",
+				"properties": {
+					"errors": {
+						"type": "array",
+						"items": {
+							"$ref": "#/components/schemas/ErrorDetailDto"
+						}
+					}
+				}
+			},
+			"ValidationErrorDto": {
+				"type": "object",
+				"properties": {
+					"traceId": {
+						"type": "string",
+						"example": "42962de0-bdb2-11ed-9397-0242ac120004"
+					},
+					"code": {
+						"type": "string",
+						"example": "6008"
+					},
+					"message": {
+						"type": "string",
+						"example": "Something is wrong with the data, please check one more time."
+					},
+					"details": {
+						"$ref": "#/components/schemas/ErrorsListDto"
+					}
+				}
+			},
+			"CompletedTaskResponse": {
+				"type": "object",
+				"properties": {
+					"id": {
+						"type": "string",
+						"example": "8a70cd6d-bdb2-11ed-9397-0242ac120004"
+					},
+					"processInstanceId": {
+						"type": "string",
+						"example": "42962de0-bdb2-11ed-9397-0242ac120004"
+					},
+					"rootProcessInstanceId": {
+						"type": "string",
+						"example": "42962de0-bdb2-11ed-9397-0242ac120004"
+					},
+					"rootProcessInstanceEnded": {
+						"type": "boolean",
+						"example": "true"
+					},
+					"variables": {
+						"type": "object",
+						"additionalProperties": {
+							"$ref": "#/components/schemas/VariableValueResponse"
+						}
+					}
+				}
+			},
+			"VariableValueResponse": {
+				"type": "object",
+				"properties": {
+					"type": {
+						"type": "string",
+						"example": "Abc"
+					},
+					"value": {
+						"type": "object",
+						"example": "Test"
+					},
+					"valueInfo": {
+						"type": "object",
+						"additionalProperties": {
+							"type": "object"
+						}
+					}
+				}
+			},
+			"ApplicantTaskResponse": {
+				"type": "object",
+				"properties": {
+					"id": {
+						"type": "string",
+						"example": "8a70cd6d-bdb2-11ed-9397-0242ac120004"
+					},
+					"taskDefinitionKey": {
+						"type": "string",
+						"example": "42962de0-bdb2-11ed-9397-0242ac120004"
+					},
+					"name": {
+						"type": "string",
+						"example": "Application xyz"
+					},
+					"assignee": {
+						"type": "string",
+						"example": "42962de0-bdb2-11ed-9397-0242ac120004"
```

Comment:
if this is an id, what data / endpoint / registry does it point to?

Reply:
similar question for "taskDefinitionKey" actually. Where are those definitions defined? And what's the difference between task definiton and process definition further below?

Reply:
I was thinking that this is an operator who is processing the applicant  file. we need to make the operator APIs. 
API for operator users:
 - get a list of services 
 - get the details of a service, including the flow (part B roles) - this would also include  a form schema ID needed for each role to be submitted + available 
 - get the list of files for a specific part B role
 - change the status of a file in a part B role :  this will require a dynamic data object that should match the needed form schema